src/updateListener.ts

```diff
@@ -0,0 +1,130 @@
+import { Disposable } from "./utils/dispose";
```

Comment:
All of your source files need copyright headers at the top, I didn't notice that before: https://docs.opensource.microsoft.com/releasing/maintain/copyright-headers/